Formatter/Pool.php

```diff
@@ -33,7 +33,7 @@ class Pool implements LoggerAwareInterface
     protected $defaultFormatter;
 
     /**
-     * @param string $defaultFormatter
+     * @param LoggerInterface will be used to report errors
```

Comment:
this one is strange. This is not a loggerInterface or atleast not by its name: `defaultFormatter`